src/client/pages/ResumesPage/index.tsx

```diff
@@ -1,23 +1,23 @@
-import { TESCUser } from '@Shared/ModelTypes';
-import { UserStatus } from '@Shared/UserStatus';
-import React from 'react';
-import { connect } from 'react-redux';
-import { showLoading, hideLoading } from 'react-redux-loading-bar';
-import { RouteComponentProps } from 'react-router';
-import { bindActionCreators } from 'redux';
-import { ApplicationDispatch, loadAllAdminEvents } from '~/actions';
-import { loadAllSponsorUsers } from '~/data/AdminApi';
-import { ApplicationState } from '~/reducers';
-import { applyResumeFilter } from '~/static/Resumes';
-
-import { replaceApplicants, replaceFiltered } from './actions';
-import ResumeList from './components/ResumeList';
-
-type RouteProps = RouteComponentProps<{
+ import { TESCUser } from '@Shared/ModelTypes';
```

Comment:
while we're looking at this file, seems like TESCUser and UserStatus aren't used anywhere in this file